src/main/scala/thurloe/service/DataAccess.scala

```diff
@@ -0,0 +1,12 @@
+package thurloe.service
+
+import scala.util.Try
+
+trait DataAccess {
+  def setKeyValuePair(keyValuePair: KeyValuePair): Try[KeyValuePair]
```

Comment:
It's a Try so that failures can be reported back. It echo's back the just-set KeyValuePair because I didn't have anything else to send back. Maybe Try[Unit] is better?